### PR TITLE
added new bot command feature

### PR DESCRIPTION
**Pull request by bestander**

### CODE REVIEW

File: bots/IssueCommands.txt [hunk 1]
```diff
@@ -31,3 +31,7 @@ close
 @facebook-github-bot reopen
 comment Okay, reopening this issue.
 reopen
+
+@facebook-github-bot feature
+comment Hey {issue_author}! Thanks for opening the issue, but we are closing it because it looks like a feature request, feel free to send us a Pool Request or create an entry on [Product Pains](https://productpains.com/product/react-native).'
```
Comment (mkonicek) [inline bots/IssueCommands.txt]: Pool request :)

Comment (bestander) [inline bots/IssueCommands.txt]: ups :)

Comment (Kureev) [inline bots/IssueCommands.txt]: :laughing:

File: bots/IssueCommands.txt [hunk 2]
```diff
@@ -31,3 +31,7 @@ close
 @facebook-github-bot reopen
 comment Okay, reopening this issue.
 reopen
+
+@facebook-github-bot feature
+comment Hey {issue_author}! Thanks for opening the issue, but we are closing it because it looks like a feature request. If this can't be implemented as a standalone [npm](https://npmjs.org) module free to send us a Pull Request or create an entry on [Product Pains](https://productpains.com/product/react-native). It has a voting system and if the feature gets upvoted enough it might get implemented.'
+reopen
```
Comment (Kureev) [inline bots/IssueCommands.txt]: Maybe we don't need a "reopen" here?

**Comment by Kureev** [comment]
LGTM :+1:

File: bots/IssueCommands.txt [hunk 3]
```diff
@@ -31,3 +31,7 @@ close
 @facebook-github-bot reopen
 comment Okay, reopening this issue.
 reopen
+
+@facebook-github-bot feature
+comment Hey {issue_author}! Thanks for opening the issue, but we are closing it because it looks like a feature request. If this can't be implemented as a standalone [npm](https://npmjs.org) module free to send us a Pull Request or create an entry on [Product Pains](https://productpains.com/product/react-native). It has a voting system and if the feature gets upvoted enough it might get implemented.'
```
Comment (mkonicek) [inline bots/IssueCommands.txt]: nit: extra `'`

"Thanks for opening the issue, however it looks like a feature request. As noted in the [Issue template](https://github.com/facebook/react-native/blob/master/ISSUE_TEMPLATE.md) we'd like to use the GitHub issues to track bugs only. Can you implement the feature as a standalone npm module? If not consider sending a pull request or a create an entry ..."

"Closing this now due to the explanation above, thanks for understanding!"

**Comment by mkonicek** [comment]
Some nits, looks good in general. I'd mention we're closing it at the end of the message, after explaining the reasons, so the person has the context first on why it's being closed.

**Comment by bestander** [comment]
Ok, will rephrase a bit in 10 min

**Comment by mkonicek** [comment]
Merging so we can start using this. Feel free to tweak later :)

@facebook-github-bot shipit

**Pull request closed without merging**